src/pages/on-chain-governance/_components/Tooltip.tsx

```diff
@@ -0,0 +1,26 @@
+import classNames from "classnames";
+import React from "react";
+
+interface TooltipProps {
+  text: string;
+  children?: JSX.Element;
+  active?: boolean;
+  customStyle?: string;
+}
+
+export function Tooltip({ text, children, active, customStyle }: TooltipProps) {
```

Comment:
let's use this common component, to have consistent behaviour `src/components/commons/popover/InfoHoverPopover.tsx`